frontend/packages/container-security/src/components/summary.tsx

```diff
@@ -37,8 +37,8 @@ export const quayURLFor = (vuln: ImageManifestVuln) => {
   return `//${base}/manifest/${vuln.spec.manifest}?tab=vulnerabilities`;
 };
 
-export const SecurityBreakdownPopup: React.FC<WatchK8sResults<WatchImageVuln>> = ({
-  imageManifestVuln,
+export const SecurityBreakdownPopup: React.FC<ResourcesPopupProps<WatchImageVuln>> = ({
```

Comment:
we could keep this file without change

and change 
https://github.com/openshift/console/blob/master/frontend/packages/console-plugin-sdk/src/typings/dashboards.ts#L95

to 

```
popupComponent?: LazyLoader<WatchK8sResults<R>>;
```

which IMO yields nicer types